studio-docs/source/explorer/sandbox.mdx

```diff
@@ -76,6 +78,17 @@ Select any accessible organization, graph, and variant to check against and clic
 
 To view the result's _full_ details in Studio, click **Go to details**.
 
+#### Subgraph checks
+
+You can run subgraph checks on federated graphs. If you are introspecting a subgraph you will see a subgraph dropdown where you can select the subgraph that you want your local subgraph to replace.
+
+<img class="screenshot" src="../img/sandbox-subgraph-check-prompt.jpg" alt="Sandbox subgraph check prompt" width="400" />
+
+When you click **Run a check**, composition validation will be run. If composition is successful, you will see both the composition result and the results of the operation check, otherwise you will just see the composition failure.  
```

Comment:
"composition validation will be run..." -> "Studio first performs a composition check using your local subgraph schema and the registered schemas of your _other_ subgraphs. If composition succeeds, Sandbox then performs operation checks as usual and displays the results of _all_ checks. If composition fails, Sandbox does _not_ perform operation checks and displays the composition error."